src/Evolution/Systems/Cce/Actions/InitializeWorldtubeBoundary.hpp

```diff
@@ -9,31 +9,94 @@
 
 #include "DataStructures/DataBox/DataBox.hpp"
 #include "DataStructures/VariablesTag.hpp"
+#include "Evolution/Systems/Cce/AnalyticBoundaryDataManager.hpp"
 #include "Evolution/Systems/Cce/BoundaryData.hpp"
 #include "Evolution/Systems/Cce/InterfaceManagers/GhInterfaceManager.hpp"
+#include "Evolution/Systems/Cce/InterfaceManagers/GhLocalTimeStepping.hpp"
 #include "Evolution/Systems/Cce/InterfaceManagers/GhLockstep.hpp"
 #include "Evolution/Systems/Cce/OptionTags.hpp"
 #include "Evolution/Systems/Cce/Tags.hpp"
-#include "Evolution/Systems/Cce/WorldtubeDataManager.hpp"
 #include "NumericalAlgorithms/Interpolation/SpanInterpolator.hpp"
 #include "Parallel/Algorithm.hpp"
 #include "Parallel/Info.hpp"
 #include "Parallel/Invoke.hpp"
 #include "ParallelAlgorithms/Initialization/MutateAssign.hpp"
 #include "Utilities/Gsl.hpp"
+#include "Utilities/MakeString.hpp"
 #include "Utilities/Requires.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
 
 namespace Cce {
+/// \cond
+template <typename Metavariables>
+struct H5WorldtubeBoundary;
+template <typename Metavariables>
+struct AnalyticWorldtubeBoundary;
+template <typename Metavariables>
+struct GhWorldtubeBoundary;
+/// \endcond
 namespace Actions {
+
+namespace detail {
+template <typename Initializer, typename ManagerTag, typename BoundaryCom>
+struct InitializeWorldtubeBoundaryBase;
+
+template <typename WorldtubeComponent, template <typename> class Initializer,
```

Comment:
don't define a specialization, there is no use of `WorldtubeComponent`.  (i.e. remove lines 44-48)